src/main/java/org/sagebionetworks/bridge/reporter/request/ReportType.java

```diff
@@ -5,7 +5,8 @@
 public enum ReportType {
     DAILY("DAILY"), // aka DAILY_UPLOADS 
     WEEKLY("WEEKLY"), // aka WEEKLY_UPLOADS
-    DAILY_SIGNUPS("DAILY_SIGNUPS");
+    DAILY_SIGNUPS("DAILY_SIGNUPS"),
+	DAILY_RETENTION("DAILY_RETENTION");
```

Comment:
Replace tab with four spaces. Four spaces per indentation level is the standard for all Bridge java code.